readme.md

```diff
@@ -56,6 +54,27 @@ From an npm script in package.json. (Requires rokudeploy.json to exist at the ro
         }
     }
 
+A callback has also been added to the higher level methods so you can modify the copied contents before the package is zipped. An info object is passed in with the following attributes
```

Comment:
Change the wording a little bit. "has been added" makes this feel like a changelog, when in reality, people are coming here to read how to do it. So something like "You can provide a callback in any of the higher level methods, which allow you do things like modify.....

Reply:
Makes sense. That's what happens when you're doing it at the same time you're writing commits and PRs will change in next commit.